README.md

```diff
@@ -52,13 +52,19 @@ This is a work-in-progress implementation of a service broker for services provi
 `PASSWORD` - username password
 `DB_PREFIX` - a prefix that can be used to tag instances. Defaults to `compose-broker`
 `CLUSTER_NAME` - a name of your enterprise cluster if you've got one and want to use it. Defaults to hosted compose
-`ACCESS_TOKEN` - your API key for Compose.
+`COMPOSE_ACCESS_TOKEN` - your API key for Compose.
```

Comment:
Just for clarity. I had an ACCESS_TOKEN in environment already which caused momentary confusion.